app/backend/src/couchers/errors.py

```diff
@@ -22,6 +22,7 @@
 CANT_INVITE_SELF = "You can't invite yourself to a chat."
 CANT_INVITE_TO_DM = "You can't invite other users to a direct message."
 CANT_MAKE_SELF_ADMIN = "You can't make yourself an admin."
+CANT_MESSAGE_IN_CHAT = "You can't message in this chat."
```

Comment:
"You can't send a message in this chat."

Reply:
Done 👍